utils/vlog/vlog_to_pbtype.py

```diff
@@ -359,27 +599,26 @@ def process_clocked_tmg(tmgspec, port, xmltype, xml_parent):
                 attrs["value"] = splitspec[1]
             ET.SubElement(xml_parent, xmltype, attrs)
 
-    # Process timing
     for name, width, bits, iodir in mod.ports:
-        port = "{}.{}".format(mod_pname, name)
+        port = "{}".format(name)
         # Clocked timing
         Tsetup = mod.net_attr(name, "SETUP")
         Thold = mod.net_attr(name, "HOLD")
         Tctoq = mod.net_attr(name, "CLK_TO_Q")
-        process_clocked_tmg(Tsetup, port, "T_setup", xml_parent)
-        process_clocked_tmg(Thold, port, "T_hold", xml_parent)
-        process_clocked_tmg(Tctoq, port, "T_clock_to_Q", xml_parent)
+        process_clocked_tmg(Tsetup, port, "T_setup", pb_type_xml)
+        process_clocked_tmg(Thold, port, "T_hold", pb_type_xml)
+        process_clocked_tmg(Tctoq, port, "T_clock_to_Q", pb_type_xml)
```

Comment:
Only output ports have T_clock_to_Q, and only input ports have T_setup and T_hold.  I believe this is not right?

Reply:
@kgugala

Reply:
I've added assert to catch those